From 2dbee53a93d44ae21a5f14025f0322b4e1e4cc88 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 9 Jun 2020 22:17:32 +0800
Subject: [PATCH] OPTEE: fix clang error

security/optee_linuxdriver/core/tee_session.c:57:3: error: 'sprintf' will
always overflow; destination buffer has size 35, but format string expands
to at least 36 [-Werror,-Wfortify-source]

Change-Id: Ie5b2ccfc4fca053b6d88278b1019997ad0dd3ee5
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 security/optee_linuxdriver/core/tee_session.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/optee_linuxdriver/core/tee_session.c b/security/optee_linuxdriver/core/tee_session.c
index 32ffb07a131d..a12d8a08415e 100644
--- a/security/optee_linuxdriver/core/tee_session.c
+++ b/security/optee_linuxdriver/core/tee_session.c
@@ -48,7 +48,7 @@ static inline bool is_mapped_temp(int flags)
 
 /******************************************************************************/
 
-#define _UUID_STR_SIZE 35
+#define _UUID_STR_SIZE 36
 static char *_uuid_to_str(const TEEC_UUID *uuid)
 {
 	static char uuid_str[_UUID_STR_SIZE];
-- 
2.35.3

